internal/pkg/panic/panic_test.go

```diff
@@ -0,0 +1,17 @@
+package panic
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseStack(t *testing.T) {
+	rawTrace := "runtime/debug.Stack()\n        ~/.goenv/versions/1.20.4/src/runtime/debug/stack.go:24 +0x65\ngithub.com/confluentinc/cli/internal/cmd.Execute.func1()\n        ~/cli/internal/cmd/command.go:157 +0x198\npanic({0x1019356e0, 0xc0004ba990})\n        ~/.goenv/versions/1.20.4/src/runtime/panic.go:884 +0x213\ngithub.com/confluentinc/cli/internal/cmd.Execute(0xc000004600, {0xc000040050?, 0x101d1b4c0?, 0x28?}, 0xc0002c62c0)\n        ~/cli/internal/cmd/command.go:172 +0x17c\nmain.main()\n        ~/cli/cmd/confluent/main.go:40 +0x3f4\n"
```

Comment:
Can you create a raw string with backticks here, so it's easier to read this? The `\n` are distracting...